src/daemon/emitter.cc

```diff
@@ -478,10 +494,13 @@ void Emitter::DoRemoteExecute(const base::WorkerPool& pool, ResolveFn resolver,
       if (status.code() != net::proto::Status::OK) {
         LOG(WARNING) << "Remote compilation failed with error(s):" << std::endl
                      << status.description();
+        STAT(REMOTE_COMPILATION_FAILED);
         failed_tasks_->Push(std::move(*task));
         counter.ReportOnDestroy(true);
         continue;
       }
+    } else {
+      STAT(REMOTE_COMPILATION_REJECTED);
```

Comment:
There is `OVERLOAD` in status codes, but it's unused for now. I suggest to implement the overload reporting in a proper way on absorber.